2.7.0/todo/result_intel.md

```diff
@@ -30,12 +30,18 @@ The categories below are as follows:
 ### deprecation
 ### new features
 ### improvements
+- Upgrade submodule oneDNN to v3.7.1 ([#148293](https://github.com/pytorch/pytorch/pull/148293))
+- [CPU][Brgemm] add support for int8 brgemm ([#143384](https://github.com/pytorch/pytorch/pull/143384))
```

Comment:
```suggestion
- Add support for int8 brgemm (#143384)
```
Please remove starting tags [..] following the guidance in https://docs.google.com/document/d/14OmgGBr1w6gl1VO47GGGdwrIaUNr92DFhQbY_NEk8mQ/edit?tab=t.0#bookmark=id.xdg38xr12qr0

Reply:
Thanks and changed